oh-my-posh is not supported by Oh My Posh

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I updated to 24.0.2 on my Windows 10 machine. I had to change my profile script from `Invoke-Expression (oh-my-posh --init --shell pwsh --config "<<path to config>>")` to `oh-my-posh init pwsh --config "<<path to config>>" | Invoke-Expression` as per the docs. But instead of the theme, all I get is the string "oh-my-posh is not supported by Oh My Posh" on my terminal

Note, I also run WSL and the updated OMP works fine there once I changed the init command

### Theme

my own custom

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 24.0.2

Shell: powershell

Prompt:

 harry   ]8;;file:C:/Users/harry/Source/scripts\scripts]8;;\  ]8;;https://github.com/devhawk/personalScripts.git\ ]8;;\ master ↓1  ?3 ~3    7                                                                                                                   in powershell at 08:54:108

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   0 ms
Path(true)                                -   1 ms
Git(true)                                 -  55 ms
Root(false)                               -   0 ms
Status(true)                              -   0 ms
Node(false)                               -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   1 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 59.8327ms

Cache path: C:\Users\harry\AppData\Local\oh-my-posh

Config path: no --config set, using default built-in configuration

Logs:

[38;2;191;207;240m[DEBUG] [38;2;156;231;201m08:54:10.231 [0mdebug.go:50[38;2;156;231;201m →[0m debug mode enabled
[DEBUG] 08:54:10.232 debug.go:50 → plain mode enabled
[DEBUG] 08:54:10.232 terminal.go:Getenv:161 → C:\Users\harry\AppData\Local
[TRACE] 08:54:10.232 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 08:54:10.232 terminal.go:CachePath() - 0s
[DEBUG] 08:54:10.232 terminal.go:65 → loading cache file: C:\Users\harry\AppData\Local\oh-my-posh\omp.cache
[DEBUG] 08:54:10.233 terminal.go:65 → loading cache key: upgrade_segment
[TRACE] 08:54:10.233 terminal.go(C:\Users\harry\AppData\Local\oh-my-posh\omp.cache) - 519.1µs
[DEBUG] 08:54:10.233 terminal.go:Getenv:161 → C:\Users\harry\AppData\Local
[TRACE] 08:54:10.233 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 08:54:10.233 terminal.go:CachePath() - 0s
[DEBUG] 08:54:10.233 terminal.go:65 → loading cache file: C:\Users\harry\AppData\Local\oh-my-posh\omp.cache.26548
[ERROR] 08:54:10.233 terminal.go:65 → open C:\Users\harry\AppData\Local\oh-my-posh\omp.cache.26548: The system cannot find the file specified.
[TRACE] 08:54:10.233 terminal.go(C:\Users\harry\AppData\Local\oh-my-posh\omp.cache.26548) - 0s
[TRACE] 08:54:10.233 terminal.go:setPromptCount() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.233 terminal.go:setPwd:194 → C:\Users\harry\Source\scripts
[TRACE] 08:54:10.233 terminal.go:setPwd() - 0s
[DEBUG] 08:54:10.233 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.233 terminal.go:Getenv(POSH_THEME) - 0s
[DEBUG] 08:54:10.233 terminal.go:ResolveConfigPath:99 → no config set, fallback to default config
[TRACE] 08:54:10.233 terminal.go:ResolveConfigPath() - 0s
[TRACE] 08:54:10.233 terminal.go:Init() - 1.0373ms
[TRACE] 08:54:10.233 terminal.go:Shell() - 0s
[TRACE] 08:54:10.233 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.233 load.go:loadConfig:55 → no config file specified, using default
[TRACE] 08:54:10.233 load.go:loadConfig() - 0s
[TRACE] 08:54:10.233 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.233 debug.go:60 → terminal program: vscode
[DEBUG] 08:54:10.233 debug.go:60 → terminal shell: shell
[DEBUG] 08:54:10.233 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.233 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 08:54:10.233 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.233 terminal_windows.go:WindowsRegistryKeyValue:213 → ColorizationColor(DWORD): 0xC46B69D6
[TRACE] 08:54:10.233 colors_windows.go:GetAccentColor() - 0s
[TRACE] 08:54:10.233 colors.go:SetAccentColor() - 0s
[TRACE] 08:54:10.233 terminal.go:Shell() - 0s
[DEBUG] 08:54:10.233 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.233 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 08:54:10.233 debug.go:PrintDebug:23 → segment: Title
[TRACE] 08:54:10.233 terminal_windows.go:Root() - 0s
[TRACE] 08:54:10.233 terminal.go:Shell() - 0s
[TRACE] 08:54:10.233 terminal.go:StatusCodes() - 0s
[TRACE] 08:54:10.233 terminal_windows.go:IsWsl() - 0s
[DEBUG] 08:54:10.233 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal_windows.go:IsWsl() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.233 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.233 terminal.go:User:331 → harry
[TRACE] 08:54:10.233 terminal.go:User() - 0s
[DEBUG] 08:54:10.234 terminal.go:Host:348 → HLP-TARDIS
[TRACE] 08:54:10.234 terminal.go:Host() - 994.9µs
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.234 terminal.go:Getenv(SHLVL) - 0s
[TRACE] 08:54:10.234 terminal.go:TemplateCache() - 994.9µs
[TRACE] 08:54:10.234 text.go:Render({{ .Shell }} in {{ .Folder }}) - 994.9µs
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.234 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.234 terminal.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 08:54:10.234 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.234 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.234 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 segment.go:Execute:103 → segment: Status
[DEBUG] 08:54:10.234 segment.go:Execute:103 → segment: Session
[TRACE] 08:54:10.234 terminal.go:StatusCodes() - 0s
[DEBUG] 08:54:10.234 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.234 terminal.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 08:54:10.234 terminal.go:Getenv:161 → NO DATA
[TRACE] 08:54:10.234 terminal.go:Getenv(SSH_CLIENT) - 0s
[DEBUG] 08:54:10.234 status.go:formatStatus:45 → {{ .Code }}
[TRACE] 08:54:10.234 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.234 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 segment.go:Execute:103 → segment: Path
[DEBUG] 08:54:10.234 path.go:145 → display_cygpath: false
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.234 terminal.go:PathSeparator() - 0s
[TRACE] 08:54:10.234 terminal.go:Shell() - 0s
[TRACE] 08:54:10.234 terminal.go:Shell() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 path.go:setMappedLocations:552 → mapped_locations_enabled: true
[DEBUG] 08:54:10.234 path.go:setMappedLocations:553 → 
[DEBUG] 08:54:10.234 path.go:setMappedLocations:554 → 
[DEBUG] 08:54:10.234 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.234 terminal.go:PathSeparator() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.234 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 segment.go:Execute:103 → segment: Git
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.234 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.234 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.234 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.234 segment.go:Execute:103 → segment: Root
[TRACE] 08:54:10.234 text.go:Render({{ .Code }}) - 0s
[DEBUG] 08:54:10.234 status.go:Enabled:37 → always_enabled: true
[TRACE] 08:54:10.234 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.234 terminal_windows.go:Root() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.235 path.go:setMappedLocations:555 → ~
[DEBUG] 08:54:10.235 path.go:setMappedLocations:560 → mapped_locations: map[]
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:PathSeparator() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:PathSeparator() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.235 path.go:makeFolderFormatMap:827 → NO DATA
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.235 path.go:setStyle:209 → folder
[DEBUG] 08:54:10.235 path.go:colorizePath:728 → cycle: []
[DEBUG] 08:54:10.235 path.go:getFolderSeparator:263 → NO DATA
[DEBUG] 08:54:10.235 path.go:getFolderSeparator:265 → \
[DEBUG] 08:54:10.235 path.go:colorizePath:731 → cycle_folder_separator: false
[DEBUG] 08:54:10.235 path.go:colorizePath:732 → %s
[DEBUG] 08:54:10.235 path.go:colorizePath:734 → %s
[DEBUG] 08:54:10.235 path.go:colorizePath:735 → %s
[DEBUG] 08:54:10.235 path.go:colorizePath:736 → %s
[TRACE] 08:54:10.235 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.235 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.235 terminal.go:StackCount() - 0s
[TRACE] 08:54:10.235 text.go:Render( {{ if .SSHSession }} {{ end }}{{ .UserName }} ) - 1ms
[TRACE] 08:54:10.235 text.go:Render(diamond) - 0s
[DEBUG] 08:54:10.236 win32_windows.go:isWriteable:258 → not current user or in group
[DEBUG] 08:54:10.236 win32_windows.go:isWriteable:262 → current user is member of S-1-5-32-544
[DEBUG] 08:54:10.236 win32_windows.go:isWriteable:270 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 08:54:10.236 win32_windows.go:isWriteable:272 → user has write access
[TRACE] 08:54:10.236 terminal_windows.go:DirIsWritable() - 998.9µs
[TRACE] 08:54:10.236 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.236 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.236 text.go:Render(  {{ path .Path .Location }} ) - 0s
[TRACE] 08:54:10.236 text.go:Render(powerline) - 0s
[DEBUG] 08:54:10.240 terminal.go:CommandPath:395 → C:\Program Files\Git\cmd\git.exe
[TRACE] 08:54:10.240 terminal.go:CommandPath(git.exe) - 6.9268ms
[TRACE] 08:54:10.240 terminal.go:HasCommand(git.exe) - 6.9766ms
[DEBUG] 08:54:10.240 git.go:shouldDisplay:326 → fetch_bare_info: false
[DEBUG] 08:54:10.240 terminal.go:ResolveSymlink:287 → C:\Users\harry\Source\scripts
[TRACE] 08:54:10.240 terminal.go:ResolveSymlink(C:\Users\harry\Source\scripts) - 0s
[TRACE] 08:54:10.240 terminal.go:HasParentFilePath(.git) - 0s
[DEBUG] 08:54:10.240 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.240 git.go:Enabled:172 → fetch_user: false
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.240 git.go:Enabled:187 → cli
[DEBUG] 08:54:10.240 git.go:Enabled:192 → fetch_status: true
[DEBUG] 08:54:10.240 git.go:shouldIgnoreStatus:464 → ignore_status: []
[DEBUG] 08:54:10.240 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.240 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.240 git.go:setGitStatus:610 → status_formats: map[]
[DEBUG] 08:54:10.240 git.go:getSwitchMode:909 → untracked_modes: map[]
[DEBUG] 08:54:10.240 git.go:getSwitchMode:909 → ignore_submodules: map[]
[DEBUG] 08:54:10.289 terminal.go:RunCommand:371 ↓
    # branch.oid ec10553aff33f60a9afcf745eb6c9fa3ef836f2d
    # branch.head master
    # branch.upstream origin/master
    # branch.ab +0 -1
    1 .M N... 100644 100644 100644 30dde0428f5ad4453fea37d93291bf4867bcdfed 30dde0428f5ad4453fea37d93291bf4867bcdfed OhMyPosh/devhawk.omp.json
    1 .M N... 100644 100644 100644 1dee779b37b5f54b8d57cda5a9ec31193bb2528a 1dee779b37b5f54b8d57cda5a9ec31193bb2528a profile.ps1
    1 .M N... 100644 100644 100644 76e2fd358ad2c1fa535cd43d305acb0448b7eb35 76e2fd358ad2c1fa535cd43d305acb0448b7eb35 wsl/docker-install.sh
    ? OhMyPosh/devhawk.omp.json.bak
    ? log.txt
    ? update-dns.ps1
[TRACE] 08:54:10.289 terminal.go:RunCommand(git.exe -C C:/Users/harry/Source/scripts --no-optional-locks -c core.quotepath=false -c color.status=false status -unormal --branch --porcelain=2) - 48.1465ms
[DEBUG] 08:54:10.289 git.go:setGitHEADContext:669 → 
[DEBUG] 08:54:10.289 scm.go:formatBranch:106 → mapped_branches: map[]
[DEBUG] 08:54:10.289 scm.go:formatBranch:122 → full_branch_path: true
[DEBUG] 08:54:10.289 scm.go:formatBranch:128 → branch_max_length: 25
[DEBUG] 08:54:10.289 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.289 terminal.go:HasFolder(C:\Users\harry\Source\scripts\.git/rebase-merge) - 0s
[DEBUG] 08:54:10.289 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.289 terminal.go:HasFolder(C:\Users\harry\Source\scripts\.git/rebase-apply) - 0s
[DEBUG] 08:54:10.289 git.go:setGitHEADContext:720 → 
[DEBUG] 08:54:10.289 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.289 terminal.go:HasFilesInDir(MERGE_MSG) - 529.8µs
[DEBUG] 08:54:10.289 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.289 terminal.go:HasFilesInDir(CHERRY_PICK_HEAD) - 0s
[DEBUG] 08:54:10.289 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.289 terminal.go:HasFilesInDir(REVERT_HEAD) - 0s
[DEBUG] 08:54:10.289 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.289 terminal.go:HasFilesInDir(sequencer/todo) - 0s
[DEBUG] 08:54:10.289 git.go:477 → ↓
[DEBUG] 08:54:10.289 git.go:Enabled:205 → fetch_upstream_icon: true
[DEBUG] 08:54:10.289 git.go:getUpstreamIcon:547 → upstream_icons: map[]
[DEBUG] 08:54:10.289 git.go:getUpstreamIcon:568 →  
[TRACE] 08:54:10.289 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.290 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.290 text.go:Render( {{ if .UpstreamURL }}{{ url .UpstreamIcon .UpstreamURL }} {{ end }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }}  {{ .Working.String }}{{ end }}{{ if .Staging.Changed }}  {{ .Staging.String }}{{ end }} ) - 1.0853ms
[TRACE] 08:54:10.290 text.go:Render({{ if or (.Working.Changed) (.Staging.Changed) }}p:yellow{{ end }}) - 0s
[TRACE] 08:54:10.290 text.go:Render({{ if or (.Working.Changed) (.Staging.Changed) }}p:black{{ end }}) - 0s
[TRACE] 08:54:10.290 text.go:Render(powerline) - 0s
[TRACE] 08:54:10.290 text.go:Render({{ if or (.Working.Changed) (.Staging.Changed) }}p:yellow{{ end }}) - 0s
[TRACE] 08:54:10.290 text.go:Render({{ if or (.Working.Changed) (.Staging.Changed) }}p:black{{ end }}) - 0s
[TRACE] 08:54:10.290 text.go:Render(powerline) - 0s
[TRACE] 08:54:10.290 text.go:Render( {{ if gt .Code 0 }}{{ else }}{{ end }} ) - 0s
[TRACE] 08:54:10.290 text.go:Render({{ if gt .Code 0 }}p:red{{ end }}) - 0s
[TRACE] 08:54:10.290 text.go:Render(diamond) - 0s
[TRACE] 08:54:10.290 text.go:Render({{ if or (.Working.Changed) (.Staging.Changed) }}p:yellow{{ end }}) - 0s
[TRACE] 08:54:10.290 text.go:Render({{ if gt .Code 0 }}p:red{{ end }}) - 0s
[TRACE] 08:54:10.290 terminal.go:Shell() - 0s
[TRACE] 08:54:10.290 terminal.go:Shell() - 0s
[TRACE] 08:54:10.290 terminal.go:Flags() - 0s
[TRACE] 08:54:10.290 terminal.go:Flags() - 0s
[TRACE] 08:54:10.290 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.290 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.290 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.290 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.290 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.290 segment.go:Execute:103 → segment: Time
[TRACE] 08:54:10.290 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.290 segment.go:Execute:103 → segment: Go
[TRACE] 08:54:10.290 terminal.go:Flags() - 0s
[DEBUG] 08:54:10.290 time.go:Enabled:27 → 15:04:05
[DEBUG] 08:54:10.290 language.go:Enabled:117 → extensions: [*.go go.mod]
[DEBUG] 08:54:10.290 segment.go:Execute:103 → segment: Node
[TRACE] 08:54:10.290 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.290 terminal_windows.go:56 → C:\Users\harry
[DEBUG] 08:54:10.290 language.go:Enabled:118 → folders: []
[TRACE] 08:54:10.290 terminal.go:TemplateCache() - 0s
[DEBUG] 08:54:10.290 segment.go:Execute:103 → segment: Python
[DEBUG] 08:54:10.290 language.go:Enabled:125 → home_enabled: false
[DEBUG] 08:54:10.291 terminal_windows.go:56 → C:\Users\harry
[TRACE] 08:54:10.291 terminal.go:GOOS() - 0s
[DEBUG] 08:54:10.291 language.go:Enabled:137 → files
[DEBUG] 08:54:10.291 language.go:Enabled:117 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 08:54:10.291 segment.go:Execute:103 → segment: Shell
[DEBUG] 08:54:10.291 python.go:Enabled:56 → files
[DEBUG] 08:54:10.291 language.go:Enabled:125 → home_enabled: false
[DEBUG] 08:54:10.291 language.go:Enabled:117 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 08:54:10.291 terminal_windows.go:56 → C:\Users\harry
[DEBUG] 08:54:10.291 shell.go:Enabled:26 → mapped_shell_names: map[]
[DEBUG] 08:54:10.291 language.go:Enabled:118 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 08:54:10.291 language.go:Enabled:137 → files
[TRACE] 08:54:10.291 terminal.go:Shell() - 0s
[DEBUG] 08:54:10.291 language.go:Enabled:125 → home_enabled: false
[TRACE] 08:54:10.291 terminal.go:Flags() - 0s
[TRACE] 08:54:10.291 terminal.go:TemplateCache() - 0s
[DEBUG] 08:54:10.291 terminal_windows.go:56 → C:\Users\harry
[DEBUG] 08:54:10.291 node.go:loadContext:48 → fetch_package_manager: false
[DEBUG] 08:54:10.291 python.go:loadContext:64 → fetch_virtual_env: false
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(*.go) - 24.4µs
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(go.mod) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(*.py) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(*.js) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(*.ipynb) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(*.ts) - 0s
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(pyproject.toml) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(package.json) - 0s
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(venv.bak) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(.nvmrc) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(pnpm-workspace.yaml) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(.pnpmfile.cjs) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFilesInDir:243 → false
[TRACE] 08:54:10.291 terminal.go:HasFilesInDir(.vue) - 0s
[TRACE] 08:54:10.291 text.go:Render(plain) - 0s
[TRACE] 08:54:10.291 text.go:Render(plain) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.291 terminal.go:HasFolder(.venv) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.291 terminal.go:HasFolder(venv) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.291 terminal.go:HasFolder(virtualenv) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.291 terminal.go:HasFolder(venv-win) - 0s
[DEBUG] 08:54:10.291 terminal.go:HasFolder:272 → false
[TRACE] 08:54:10.291 terminal.go:HasFolder(pyenv-win) - 531.7µs
[TRACE] 08:54:10.291 text.go:Render(plain) - 0s
[TRACE] 08:54:10.291 terminal.go:TemplateCache() - 0s
[TRACE] 08:54:10.291 text.go:Render(in <p:blue><b>{{ .Name }}</b></> ) - 0s
[TRACE] 08:54:10.291 text.go:Render(plain) - 0s
[TRACE] 08:54:10.291 text.go:Render(at <p:blue><b>{{ .CurrentDate | date "15:04:05" }}</b></>) - 0s
[TRACE] 08:54:10.291 text.go:Render(plain) - 0s
[TRACE] 08:54:10.291 terminal.go:Shell() - 0s
[TRACE] 08:54:10.291 terminal.go:Shell() - 0s
[TRACE] 08:54:10.291 terminal.go:Flags() - 0s
[TRACE] 08:54:10.291 terminal.go:Shell() - 0s
[DEBUG] 08:54:10.291 terminal_windows.go:TerminalWidth:114 → terminal width: 192
[TRACE] 08:54:10.291 terminal_windows.go:TerminalWidth() - 0s
[DEBUG] 08:54:10.291 terminal.go:Getenv:161 → C:\Users\harry\AppData\Local
[TRACE] 08:54:10.291 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 08:54:10.291 terminal.go:CachePath() - 0s
[TRACE] 08:54:10.291 terminal.go:Flags() - 0s
```

--- a/src/cli/init.go
+++ b/src/cli/init.go
@@ -58,7 +58,7 @@ See the documentation to initialize your shell: https://ohmyposh.dev/docs/instal
 
 			env := &runtime.Terminal{
 				CmdFlags: &runtime.Flags{
-					Shell:  shellName,
+					Shell:  args[0],
 					Config: configFlag,
 					Strict: strict,
 					Debug:  debug,

